src/main/java/ch/njol/skript/expressions/ExprVillagerLevel.java

```diff
@@ -59,16 +75,20 @@ public void change(Event event, Object @Nullable [] delta, ChangeMode mode) {
 		for (LivingEntity livingEntity : getExpr().getArray(event)) {
 			if (!(livingEntity instanceof Villager villager)) continue;
 
-			int previousLevel = villager.getVillagerLevel();
+			int minLevel = experience ? 0 : 1;
+			int maxLevel = experience ? Integer.MAX_VALUE : 5;
+			int previousLevel = experience ? villager.getVillagerExperience() : villager.getVillagerLevel();
```

Comment:
i think this would be better as an if-else to keep related values together
also maybe rename to value since it's not necessarily level anymore